strimzi-cluster-operator-0.33.2.yaml

```diff
@@ -0,0 +1,14500 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
what is this file for? I do not see it in `release-0.32.x` together with several others.

Reply:
Sorry, that was a mistake. Should be removed now.